examples/nodejs/eventbridge/webapp/src/utils/dynamodb.ts

```diff
@@ -0,0 +1,51 @@
+import AWS from "aws-sdk";
+
+const awsAccesskeyId = import.meta.env.VITE_APP_AWS_ACCESS_KEY_ID;
+const awsSecretAccessKey = import.meta.env.VITE_APP_AWS_SECRET_ACCESS_KEY;
```

Comment:
we need to add support for AWS_SESSION_TOKEN here. If I go to the AWS SSO splash page and choose "Get access keys", it should work. (And it does if I change your code here to add SESSION_TOKEN, plus down at line 12 below.